src/Compilers/CSharp/Test/Semantic/Semantics/LookupTests.cs

```diff
@@ -1444,6 +1444,72 @@ private void TestLookupSymbolsNestedNamespaces(List<ISymbol> actual_lookupSymbol
             Assert.Contains(aliasY, actual_lookupSymbols);
         }
 
+        [Fact]
+        public void ExtensionMethodCall()
+        {
+            var source =
+@"static class E
+{
+    internal static void F(this object o)
+    {
+    }
+}
+class C
+{
+    void M()
+    {
+        /*<bind>*/this.F/*</bind>*/();
+    }
+}";
+            var compilation = CreateCompilationWithMscorlibAndSystemCore(source);
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+            compilation.VerifyDiagnostics();
+            var exprs = GetExprSyntaxList(tree);
+            var expr = GetExprSyntaxForBinding(exprs);
+            var method = (MethodSymbol)model.GetSymbolInfo(expr).Symbol;
+            var reducedFrom = method.ReducedFrom;
+            Assert.NotNull(reducedFrom);
+            Assert.Equal("E.F(object)", reducedFrom.ToDisplayString());
+        }
+
+        [WorkItem(3651)]
```

Comment:
Yes, the URL helps make sense of these numbers after we've changed bug tracking systems five more times (or have done so in the past).